tests/functional/pages/surveys/relationship/household-relationship.page.js

```diff
@@ -2,33 +2,13 @@ import QuestionPage from '../question.page'
 
 class HouseholdRelationshipPage extends QuestionPage {
 
-  getRelationshipLabelAt(index) {
-    const elementId = browser.elements('[data-qa="relationship-title"]').value[index]
-    return browser.elementIdText(elementId.ELEMENT).value
-  }
-
   setHusbandOrWifeRelationship(index, relationshipIndex) {
-    const id = this.buildRelationshipAnswerId(index, 0)
-    browser.waitForExist(id)
-    browser.element(id).click().pause(300)
+    browser.selectByValue('#who-is-related-0', 'Husband or wife')
```

Comment:
Shouldn't these methods also be updated to be using the instance, value arguments as per the other relationship page?